gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -113,6 +113,33 @@ public abstract class AnvilExtension @Inject constructor(
   public val trackSourceFiles: Property<Boolean> = objects.property(Boolean::class.java)
     .conventionFromProperty("com.squareup.anvil.trackSourceFiles", false)
 
+  /**
+   * Enables the new KSP backend for Anvil _source generation_. This is an experimental feature that
```

Comment:
>  _source generation_.

I'm not clear on the distinction between these two options.  Technically, everything Anvil generates is source.

Reply:
IR merging doesn't generate source? Unless you count in-memory IR or its downstream compiled bytecode as source 😅

Reply:
> in-memory IR

I guess I do?  I've never really questioned that. 🤷 

> compiled bytecode

no :)